src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -882,12 +994,54 @@ public LocalDate fromString(String string) {
     tableViewCandidates.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
     tableViewCandidates.setEditable(true);
 
-    choiceTiebreakMode.getItems().addAll(TieBreakMode.values());
-    choiceTiebreakMode.getItems().remove(TieBreakMode.MODE_UNKNOWN);
+    clearAndDisableWinningRuleFields();
+    choiceTiebreakMode.getItems().addAll(TIEBREAK_MODE_VALUES);
+    choiceTiebreakMode.setOnAction(event -> {
+      clearAndDisableTiebreakFields();
+      String mode = getChoiceElse(choiceTiebreakMode, TieBreakMode.MODE_UNKNOWN);
+      switch (mode) {
+        case "Random", "Previous round counts (then random)", "Generate permutation" -> textFieldRandomSeed
```

Comment:
Here and below -- same concern I raised above about having the same string literals repeated several times in the code. We should pull these out into constants.

Reply:
Yep yep, will do.